render.js

```diff
@@ -0,0 +1,50 @@
+// render :: (string, string) -> Node
+function render(domId, content) {
+  const el = document.getElementById(domId).innerHTML = content;
```

Comment:
consider this:
```js
const dom = document.getElementById(domId)
dom.innerHTML = content

return dom
```

Reply:
Lol

Reply:
I actually don't really understand the purpose of this render function xD
`const el = document.getElementById(domId).innerHTML = content;` is wtf?